inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/config.cpp

```diff
@@ -30,11 +30,6 @@ namespace {
             {{InferenceEngine::MultiDeviceConfigParams::KEY_MULTI_DEVICE_PRIORITIES , CommonTestUtils::DEVICE_CPU}}
     };
 
-    const std::vector<std::map<std::string, std::string>> AutoConfigs = {
```

Comment:
shall you move the tested config values from AutoConfigs to MultiConfigs rather than simply remove?

Reply:
Yes, the values are already in MultiConfigs. Most AUTO tests will use MultiConfigs, but I notice some tests are deleted by mistake, I will add back.